e2e/test/scenarios/embedding/embedding-dashboard.cy.spec.js

```diff
@@ -645,6 +655,33 @@ describe("scenarios > embedding > dashboard parameters with defaults", () => {
       .findByText("There was a problem displaying this chart.")
       .should("be.visible");
   });
+
+  it("locked parameters should still render results in the preview by default (metabase#47570)", () => {
```

Comment:
This is a new test that actually opens the static embed modal. By default the preview or the actual iframe being rendered in this case should show some results since we already default the locked parameter value to `[]`. You can see the the test above test by using `null` we won't get any results.